db/migrate/20230830115531_remove_timeline_events_with_missing_target.rb

```diff
@@ -0,0 +1,11 @@
+class RemoveTimelineEventsWithMissingTarget < ActiveRecord::Migration[6.1]
+  def up
```

Comment:
Whenever we write data migrations like these, it's good practice to re-declare the models and the association just enough to run the code in the migration. 

This makes the migration independent and immune to issues related to changes in the actual models in the codebase in the future. 

Here is an example: https://github.com/pupilfirst/pupilfirst/blob/master/db/migrate/20230508125650_set_target_milestone_from_target_group.rb

In this case, you just have to define the association between `TimelineEvent` and `Target` models alone.